mmedit/datasets/pipelines/generate_assistant.py

```diff
@@ -5,6 +5,78 @@
 from .utils import make_coord
 
 
+@PIPELINES.register_module()
+class GenerateHeatmap:
+    """Generate heatmap from keypoint.
+
+    Args:
+        keypoint (str): Key of keypoint in dict.
+        ori_size (int | Tuple[int]): Original image size of keypoint.
+        target_size (int | Tuple[int]): Target size of heatmap.
+        sigma (float): Sigma parameter of heatmap. Default: 1.0
+    """
+
+    def __init__(self, keypoint, ori_size, target_size, sigma=1.0):
+        if isinstance(ori_size, int):
+            ori_size = (ori_size, ori_size)
+        else:
+            ori_size = ori_size[:2]
+        if isinstance(target_size, int):
+            target_size = (target_size, target_size)
+        else:
+            target_size = target_size[:2]
+        self.rate = (target_size[0] / ori_size[0],
```

Comment:
What does rate mean here

Reply:
Resize heatmap

Reply:
Okay should be ratio. Choose `target_ori_ratio`/`size_ratio`/...